web/src/components/merchant/listing-details/SendItemModal.tsx

```diff
@@ -63,21 +64,27 @@ const IndicateButton = styled(RoundedButton)`
   width: 370px;
 `;
 
+const indicateItemSent = async (commitId: number) => {
+  await completeCommit(commitId);
+  window.location.reload(false);
+  // TODO(#200): Update paidCommits state instead of force refresh to show that
```

Comment:
this comment should put above force refresh statement

Reply:
@ifhuang ohh okay!